var/spack/repos/builtin/packages/ffb/package.py

```diff
@@ -0,0 +1,133 @@
+        m.filter(r'FOPT\s*=.*$', 'FOPT = {0}\n'.format(fflags))
+        m.filter(r'INCDIR\s*=.*$', 'INCDIR = {0}\n'
+                 .format(spec['mpi'].headers.directories[0]))
+        m.filter(r'LIBDIR\s*=.*$', 'LIBDIR = {0}\n'
+                 .format(spec['mpi'].libs.directories[0]))
+
+        srcdir = join_path('lib', 'src')
+        utildir = join_path(workdir, 'util')
+        with open(join_path('make', 'Makeall'), 'w') as m:
+            m.write('#!/bin/csh -f\n')
+            m.write('setenv LES3DHOME {0}\n'.format(workdir))
+            m.write('cd {0}\n'.format(srcdir))
+            m.write('./Makeall\n')
+            m.write('cd {0}\n'.format(utildir))
+            m.write('./Makeall\n')
+
+        makeall = join_path('lib', 'src', 'dd_mpi', 'Makeall')
+        dd_mpi_dir = join_path('lib', 'src', 'dd_mpi')
+        with open(makeall, 'w') as m:
+            m.write('#!/bin/csh -f\n')
+            m.write('setenv LES3DHOME {0}\n'.format(workdir))
+            m.write('cd {0}\n'.format(dd_mpi_dir))
+            m.write('make lib\n')
+        os.chmod(makeall, 0o755)
+
+        makeall = join_path('.',  'Makeall.les')
+        les3d_dir = join_path('util', 'les3d.mpi')
+        les3c_dir = join_path('util', 'les3c.mpi')
+        les3ct_dir = join_path('util', 'les3ct.mpi')
+        les3x_dir = join_path('util', 'les3x.mpi')
+        with open(makeall, 'w') as m:
+            m.write('#!/bin/csh -f\n')
+            m.write('setenv LES3DHOME {0}\n'.format(workdir))
+            m.write('cd {0}\n'.format(join_path(workdir, les3d_dir)))
+            m.write('make CCOM={0}'.format(spec['mpi'].mpicc))
+            m.write(' FCOM={0}\n'.format(spec['mpi'].mpifc))
+            m.write('cd {0}\n'.format(join_path(workdir, les3c_dir)))
+            m.write('make CCOM={0}'.format(spec['mpi'].mpicc))
+            m.write(' FCOM={0}\n'.format(spec['mpi'].mpifc))
+            m.write('cd {0}\n'.format(join_path(workdir, les3ct_dir)))
+            m.write('make CCOM={0}'.format(spec['mpi'].mpicc))
+            m.write(' FCOM={0}\n'.format(spec['mpi'].mpifc))
+            m.write('cd {0}\n'.format(join_path(workdir, les3x_dir)))
+            m.write('make CCOM={0}'.format(spec['mpi'].mpicc))
+            m.write(' FCOM={0}\n'.format(spec['mpi'].mpifc))
+
+            for d in [les3c_dir, les3ct_dir, les3d_dir]:
+                editfile = join_path(d, 'FILES')
+                m = FileFilter(editfile)
+                m.filter(r'-lmpi_f77', '')
+        os.chmod(makeall, 0o755)
+
+        editfile = join_path('lib', 'src', 'REVOCAP_Refiner-0.4.3', 'OPTIONS')
+        m = FileFilter(editfile)
+        m.filter(r'ARCH\s*=.*$', 'ARCH= $(shell arch)-linux')
+        m.filter(r'CC\s*=.*$', 'CC={0}'.format(spack_cc))
+        m.filter(r'CFLAGS\s*=.*$', 'CFLAGS={0}'.format(cflags))
+        m.filter(r'CXX\s*=.*$',  'CXX={0}'.format(spack_cxx))
+        m.filter(r'CXXFLAGS\s*=.*$',
+                 'CXXFLAGS={0}'.format(' '.join(cxxflags)))
```

Comment:
If you make the change I outlined above, this is the other spot you would need to change to match it.
```suggestion
                 'CXXFLAGS={0}'.format(cxxflags))
```

Reply:
I agree with you.